frontend/src/hooks/usePlausibleTracker.ts

```diff
@@ -66,7 +66,10 @@ export type CustomEvents =
     | 'new-in-unleash-click'
     | 'new-in-unleash-dismiss'
     | 'search-opened'
-    | 'events-exported';
+    | 'events-exported'
+    | 'event-timeline-click'
```

Comment:
Maybe this one is a bit more obvious than click?
```suggestion
    | 'event-timeline-open'
```